src/main/java/com/alex/eshop/restcontroller/CategoryController.java

```diff
@@ -19,8 +17,11 @@ public CategoryController(CategoryService categoryService) {
     }
 
     @GetMapping("/categories")
-    public List<CategoryDTO> allCategoryList() {
-        return categoryService.getAllCategories();
+    public List<CategoryDTO> allCategoryList(@RequestParam(value = "page", defaultValue = "0") int page,
```

Comment:
You don't need separate **page**, **size**, **sort** parameters. Use only **Pageable pageable**. It includes all these params